New Role Suggestions - Colour Faction Roles

TheEighthDeadlySin opened the pull request:
Apparently renaming the branch deleted the pull request... At the moment, the only thing I have changed due to Luffy's thoughts is the Jester's respawn time (increased from 2 to 4). Sorry about the earlier pull request disappearing...

TheEighthDeadlySin commented:
I am not sure whether Jester should also have any attacking items, such as Bombs, that are planted on their target redirected to them. So for now, I have only included that Jester will only redirect abilities, and items used on their target will not be redirected (so Bombs, Poison, Antidote etc. will bypass this ability).

Topkinsme reviewed (changes requested):
Warlord sounds good, but has the same problems Luffy earlier mentioned, with it just being killer but different. Also, do not have a "Winning Point" statement, just make it another point

Jester sounds good, but it should not be phase end, it should be that they pick a target during the day and it works for the entire next night.

Mage sounds good, But I recommend changing all instances of Basic Prot to weak prot, as it might be confusing if we add an actual basic prot status

Siege_warrior sounds good, however I feel like it should be a day action, as in the siege is planned during the day and executed during the night instead of it being instant.

Princess sounds good

Maybe we could drop sacrificial lamb's respawn, as it's only a 1-time thing.

TheEighthDeadlySin commented:
Okay, I shall implement the changes right now!

TheEighthDeadlySin commented:
Changes implemented:
Jester: Action is now performed during the day, any item or ability used throughout the night (includes immediate actions) on their target will instead be redirected to the Jester.
Mage: Renamed Basic Protection to Weak Protection.
Siege Warrior: Changed to action is performed during the day, and its effects are applied the following night.
Sacrificial Lamb: Decreased Respawn Time from 6 to 4.

Topkinsme reviewed (changes requested):
Jester shouldn't redirect items, I only meant it should last the entire night instead of being an instant action

Everything else is good

TheEighthDeadlySin commented:
Okay, I will change Jester so only abilities (both immediate and end phase) are redirected. Items will not be affected.

TheEighthDeadlySin commented:
Removed Warlord

TheEighthDeadlySin commented:
I have added two Solo Chaos Role Suggestions: Gambler and Bounty Hunter. I have also checked to make sure all descriptions have been written in the third person.

Topkinsme commented:
I'd recommend you make a new pull req with all the solo roles like Warlord, Bounty Hunter, and gambler, since we're close to approving the other roles and we could implement the other roles and not worry about the solos for now

TheEighthDeadlySin commented:
Okay, I shall create a new role suggestion pull request containing the three solo chaos roles.

TheEighthDeadlySin commented:
Removed non-approved Solo Suggestions to a new pull request: https://github.com/Topkinsme/Color-Battle-Roles/pull/33

Topkinsme commented on strong_power/jester:
```diff
@@ -0,0 +1,10 @@
+  Also may not choose to redirect on the same player for two nights in a row.
+  Action has no cooldown.
+  Redirection ability will occur throughout the night. 
+  Only abilities (noth immediate and end phase attacks) on their target will be redirected to the Jester.
```
"not" not "noth" typo

TheEighthDeadlySin replied:
oops XD

Topkinsme reviewed (approved):
Looks good to me, except that 1 typo

Topkinsme commented:
Also, you can change Sacrificial Limb respawn to 2 phases if you want, as Luffy suggested.

TheEighthDeadlySin commented on weak_power/sacrificial_lamb:
```diff
@@ -2,4 +2,4 @@
 - The Passive Ability will only activate once.
 - No Actions, only a Passive Ability (explained above).
-- Respawns in 6 phases.
+- Respawns in 4 phases.
```
Okay.

TheEighthDeadlySin replied:
I will decrease the respawn time to 2 phases.

Topkinsme replied:
Agrred

TheEighthDeadlySin replied:
Done!

TheEighthDeadlySin commented on solo_chaos/bounty_hunter:
```diff
@@ -1,12 +1,12 @@
-  If you kill a King, you will instead receive $500 of a bounty reward.
-  Every attack will be a Weak Attack. [Until the Player purchases the item that changes from Weak to Strong Attacks.]
-  May buy two different items, which will deduct from the net profit of bounty rewards [may not purchase if net profit below price]:
+- Every successful kill will give them $200 of a bounty reward.
```
The amount earned has to be from bounty killing, it does not count money earned from other sources, such as forge.

Topkinsme commented on strong_power/jester:
```diff
@@ -3,6 +3,8 @@
-  Redirection ability will occur throughout the night. Any items and abilities (including instant attacks) on their target will be redirected to the Jester.
+  Redirection ability will occur throughout the night. 
+  Only abilities (noth immediate and end phase attacks) on their target will be redirected to the Jester.
+  Items used on their target will not be affected.
```
Items don't work, so I don't think this applies

TheEighthDeadlySin replied:
I put it in as a clarification (in case people thought Jester will also alter item usage).

Topkinsme reviewed (approved):
All good

Topkinsme commented:
I'm approving this pull request, this looks good to me.
